tools/codegen/codespec/merge_attributes.go

```diff
@@ -2,6 +2,7 @@ package codespec
 
 import "sort"
 
+// TODO: update to infer computability of inner nested attributes
```

Comment:
Looks like this might be done already?

Reply:
On nested levels the logic should be revisited